atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/sweep/queue/SweepQueueDeleter.java

```diff
@@ -68,12 +69,11 @@ public void sweep(Collection<WriteInfo> writes, Sweeper sweeper) {
                                 kvs.deleteAllTimestamps(entry.getKey(), maxTimestampByCellPartition, true);
                             }
                         });
-            } catch (IllegalArgumentException e) {
+            } catch (IllegalArgumentException | PalantirSqlException e) {
```

Comment:
Did you track down why you ran into a `PalantirSqlException`? I'd still like to clean this up, so it would be good to note where this was thrown.

Reply:
Yeah, it was a OracleSqlException which got wrapped by a VerboseSQLException which got wrapped by a PalantirSqlException. The illegal argument exception was another way this could happen, through the table remapping